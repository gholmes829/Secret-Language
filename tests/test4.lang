// basic conditionals

num main() {
    if true {
        print(0)
    }

    if false {
        print(1)
    }
    else if false {
        print(2)
    }
    else if true {
        print(3)
    }
    else if false {
        print(4)
    }
    else {
        print(5)
    }

    if false {
        print(6)
    }
    else {
        print(7)
    }

    if false {
        print(8)
    }
    else if false {
        print(9)
    }
    else {
        print(10)
    }


}