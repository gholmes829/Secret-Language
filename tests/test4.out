0.0
3.0
7.0
10.0
